src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -983,19 +983,6 @@ private static void VerifyModelForOutVarInNotExecutableCode(SemanticModel model,
             Assert.True(model.LookupNames(decl.SpanStart).Contains(decl.Identifier().ValueText));
 
             var local = (SourceLocalSymbol)symbol;
-            var typeSyntax = decl.Type;
```

Comment:
This code got moved to AssertInfoForDeclarationExpressionSyntax, which is called below.